src/pages/dashboard/hooks/useDashboardData.ts

```diff
@@ -318,7 +325,7 @@ export const useDashboardData = (filters: any) => {
     projectLoaded,
     coverImage,
     topProject,
-    refetchTotalSectionHeader,
+    refetchTotalSectionHeader: totalSectionHeaderData?.refetch ?? (() => {}),
```

Comment:
This should not be needed, unless you're expecting the data in the BE to change and the user can request a refresh? If the filter params change, the connection should notice that the requested data is different and refetch automatically. If that's not working, then let's fix up the connection implementation so it does.